client/python/lib/ovmsclient/tfs_compat/grpc/tensors.py

```diff
@@ -61,6 +63,39 @@ def make_tensor_proto(values, dtype=None, shape=None):
 
     raise NotImplementedError
 
+def as_numpy_dtype(tensor_dtype):
```

Comment:
```suggestion
def _as_numpy_dtype(tensor_dtype):
```

For all methods not exposed to the user, please start the names with `_` to indicate they're private and should not participate in generating documentation.